Cargo.toml

```diff
@@ -167,6 +173,8 @@ bench = false
 
 [lib]
 bench = false
+# cdylib is required for wasm32-unknown-unknown
```

Comment:
This is potentially annoying, https://github.com/rustwasm/wasm-pack/issues/1297 should fix it.